packages/viem-account-ledger/tsconfig.json

```diff
@@ -3,8 +3,6 @@
   "compilerOptions": {
     "rootDir": "src",
     "outDir": "lib",
-    "moduleResolution": "Node16",
```

Comment:
can you share about this? (these configs changes can be confusing)

Reply:
I wanted to export this lib in both commonjs (lib folder) and well as esm (lib-es), and this was seemingly necessary to achieve that, happy to modify if you have a better config

This was inspired by the `@ledgerhq/hw-app-eth` configs because I know it works, but happy to find better resources